packages/paste-core/components/form/src/shared/FormControlWrapper.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {FieldWrapper} from './FieldWrapper';
+import {Prefix} from './Prefix';
+import {Suffix} from './Suffix';
+import {safelySpreadFormControlProps} from './Utils';
+
+interface FormControlWrapperProps {
+  children: NonNullable<React.ReactNode>;
+  disabled?: boolean;
+  hasError?: boolean;
+  insertAfter?: React.ReactNode;
+  insertBefore?: React.ReactNode;
+}
+
+const FormControlWrapper: React.FC<FormControlWrapperProps> = ({
+  children,
+  disabled,
+  hasError,
+  insertAfter,
+  insertBefore,
+  ...props
+}) => (
+  <FieldWrapper {...safelySpreadFormControlProps(props)} disabled={disabled} hasError={hasError}>
```

Comment:
I thought we got rid of the need for this specific form control spread 🤔

Reply:
Oh I see. We should just use `{...restrictedProps}` like Select. We'll need to go back and refactor Input and Textarea.